### PR TITLE
MSI: Hcp

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-hypershift-GA-stage-ci-rosa-ci-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-hypershift-GA-stage-ci-rosa-ci-hypershift

**Comment by rnetser** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-hypershift-GA-stage-ci-rosa-ci-hypershift

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-operator__v4.13-hypershift-GA-stage-ci.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    env:
+      CHANNEL_GROUP: candidate
+      OPENSHIFT_VERSION: "4.14"
+      REGION: us-west-2
+    workflow: rosa-aws-sts-hypershift
+zz_generated_metadata:
+  branch: rhods-operator
+  org: CSPI-QE
+  repo: MSI
+  variant: v4.13-hypershift-GA-stage-ci
```
Comment (myakove) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-operator__v4.13-hypershift-GA-stage-ci.yaml:36]: All the above OCP versions are 4.14, should this be 4.14 as well?

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-operator__v4.13-hypershift-GA-stage-ci.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      memory: 200Mi
+tests:
+- as: rosa-ci-hypershift
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      CHANNEL_GROUP: candidate
+      OPENSHIFT_VERSION: "4.14"
+      REGION: us-west-2
+    workflow: rosa-aws-sts-hypershift
```
Comment (myakove) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-operator__v4.13-hypershift-GA-stage-ci.yaml:31]: Don't we want to use here openshift-cli-installer?

**Pull request closed without merging**